content/docs/integrations/kubernetes/index.md

```diff
@@ -0,0 +1,165 @@
+---
+
+![Full active cluster on roadie](./active.png)
+
+> ⚠️ Note, the Kubernetes plugin is only supported with AWS clusters. ⚠️
+> GKE support is coming soon.
+
+# AWS
+
+## Introduction
+
+In order to use the Kubernetes plugin, Roadie needs:
+ * An Assumed Role to fetch the resources from your cluster
+ * The name of your cluster
+ * URL of the loadbalancer/entry point to your Kubernetes cluster
+
+
+These are set within Roadie at the following url:
+
+```text
+https://<tenant-name>.roadie.so/administration/settings/kubernetes
+```
+
+This page describes how to create and set up the API token.
+
+## Steps
+
+In this section we will create an AWS role that will allow the roadie user to access (read only) your Kubernetes cluster's resources.
+ * [Assuming Role](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html)
+ * [Cross Federation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_common-scenarios_third-party.html)
+ * [Kubernetes RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
+ * [Trust relationship](https://aws.amazon.com/en/blogs/security/how-to-use-trust-policies-with-iam-roles/)
+
+### Step 1: Creating the cross account federation role
+
+1. Sign into your AWS console and navigate to the [IAM service](https://console.aws.amazon.com/iam/home#/home).
+
+2. Click on ”Role” link (this should be on the left handside of your screen).
+
+3. Click on the ”Create Role” button.
+
+4. Click on ”Another AWS Account” and add the account number on on the Kubernetes configuration page (in Roadie) and then click on ”Next: permissions”.
+
+![Another AWS Account](./role-creation.png)
+
+5. Ignore attached policies and click on ”Next: tags” (Roadie does not need to read your AWS resources, only access to your Kubernetes cluster).
+
+Optional: Add a tag, Key: `3rdPartyIntegration` Value: `Roadie`
+
+6. Click ”Next Review”
+
+7. For the ”Role Name” enter: ”YOUR-COMPANY-NAME-backstage-backend-role-kubernetes”
+8. For the ”Role description” enter 
+
+```
+This is a role that will be assumed by roadie to gather information on our Kubernetes clusters.
+```
+It should look like this
+
+![role-confirmation](./role-confirmation.png)
+
+9. Click ”Create role”. Your cross federation role is now created.
+
+### Step 2: Modifying trust relationships to only include the new role
+
+1. Search for IAM in the services box and then click on ”Roles” on the left handside tab.
+
+2. Search for your newly created role: ”YOUR-COMPANY-NAME-backstage-backend-role-kubernetes” and click on it.
+
+You should see a page like this
+
+![role-page](./role-page.png)
+
+3. Click on ”Trust Relationships”, then ”Edit relationship” and add the text below:
+
+``` json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "<ROLE FROM CONFIGURATION PAGE>"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+```
+
```

Comment:
I think even thought it might look obvious to us it is usually helpful to add something like:

> In the Json fragment above, replace the "<ROLE FROM CONFIGURATION PAGE>" with the role that ....blahblah... mentioned previously. In this example that would be `X` 

And then put the example string that we used in a previous step. That way the user can easily double check in their case that both things match for them too.

Reply:
Done